libpkg/repo/binary/update.c

```diff
@@ -624,8 +624,8 @@ pkg_repo_binary_update(struct pkg_repo *repo, bool force)
 
 		snprintf(filepath, sizeof(filepath), "%s/%s", ctx.dbdir,
 			pkg_repo_binary_get_filename(pkg_repo_name(repo)));
-		if (stat(filepath, &st) != -1) {
-			if (!got_meta && !force)
+		if (got_meta && stat(filepath, &st) != -1) {
+			if (!force)
```

Comment:
am i reading this right, that --force will proceed even without the meta file?

Reply:
--force leaves the timestamp 0 so that pkg considers everything out of date and refetches regardless